config/upstart/grr-dataserver-master.conf

```diff
@@ -10,11 +10,15 @@ kill timeout 300
 
 pre-start script
 	mkdir -p /var/log/grr
+	mkdir -p /var/run/grr/tmp/dataserver
```

Comment:
Lets add -master and -slave to all of these paths. It's possible (but not super useful) to run them on the same machine.